spec/factories/dogs.rb

```diff
@@ -1,8 +1,8 @@
 FactoryBot.define do
   factory :dog do
     Faker::Config.locale = "en-US"
-    name { Faker::Creature.Dog.name }
-    breed { Faker::Creature.Dog.breed }
-    age { Faker::Number.number.between(from: 1, to: 15) }
+    name { Faker::Creature::Dog.name }
```

Comment:
I had changed this too but the fetch dog tests failed with the change. I'm happy with it either way.